source/client/Controller.py

```diff
@@ -125,7 +125,10 @@ def getHand(self):
     def getPreparedCards(self):
         """lets the UI fetch prepared cards"""
         #TODO: Sheri - should this information be returned in getHand or is a separate method best?
```

Comment:
You can kill this TODO as soon as you are happy with the setup. Since you edited it I assume you are ok with it.